libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -270,7 +270,7 @@ pub struct Connector {
 
 impl Connector {
     pub fn request_connection(&self, endpoint: &str) -> Result<CmResponse, ConnectionManagerError> {
-        let (sender, recv) = bounded(1);
+        let (sender, recv) = channel();
```

Comment:
I think `sync_channel` would be the more correct change, as that has a bound.

Reply:
Changed over to sync_channel